src/main/java/minicraft/screen/KeyInputDisplay.java

```diff
@@ -10,48 +13,48 @@
 import minicraft.screen.entry.StringEntry;
 
 public class KeyInputDisplay extends Display {
-	
+
 	private boolean listeningForBind, confirmReset;
-	
+
 	private static Menu.Builder builder;
-	
+
 	private static KeyInputEntry[] getEntries() {
 		String[] prefs = Game.input.getKeyPrefs();
 		KeyInputEntry[] entries = new KeyInputEntry[prefs.length];
-		
+
 		for (int i = 0; i < entries.length; i++)
 			entries[i] = new KeyInputEntry(prefs[i]);
-		
+
 		return entries;
 	}
-	
+
 	public KeyInputDisplay() {
 		super(true);
 		builder = new Menu.Builder(false, 0, RelPos.CENTER, getEntries())
 			.setTitle("Controls")
 			.setPositioning(new Point(Screen.w/2, Screen.h - Font.textHeight()*5), RelPos.TOP);
-		
+
 		Menu.Builder popupBuilder = new Menu.Builder(true, 4, RelPos.CENTER)
 			.setShouldRender(false)
 			.setSelectable(false);
-		
+
 		menus = new Menu[] {
 			builder.createMenu(),
-			
+
 			popupBuilder
 				.setEntries(StringEntry.useLines(Color.YELLOW, "Press the desired", "key sequence"))
 				.createMenu(),
-			
+
 			popupBuilder
-				.setEntries(StringEntry.useLines(Color.RED, "Are you sure you want to reset all key bindings to the default keys?", "enter to confirm", "escape to cancel"))
+				.setEntries(StringEntry.useLines(Color.RED, "Confirm resetting key bindings.1", "Confirm resetting key bindings.2", "Confirm resetting key bindings.3", "enter to confirm", "escape to cancel"))
```

Comment:
The loc key for this should be `minicraft.display.key_input.confirm_popup`, and there should only be one entry.

Reply:
We need to separate it for line breaking.